def raise_exception():
# Call the function to see the exception
    try:
        raise_exception()
    except TypeError as te:
        print("Exception raised")
